conda_index/cli/__init__.py

```diff
@@ -0,0 +1,96 @@
+"""
+Updated command line interface for conda-index.
+"""
+import logging
+import os.path
+
+import click
+
+from conda_index.index import MAX_THREADS_DEFAULT, ChannelIndex, logutil
+from conda_index.utils import DEFAULT_SUBDIRS
+
+
+@click.command()
+@click.argument("dir")
+@click.option("--repodata", help="Output repodata to given directory")
```

Comment:
@barabo you had suggested `--output`? I changed it here, but forgot to change the parameter name in the function.

Reply:
I do prefer `--output` or `--output-dir` or `--write-to`, even.  To me, `--repodata` does not suggest whether it's used as input to the process or whether it's an output destination.